module/herosystem6e.mjs

```diff
@@ -209,7 +214,15 @@ Hooks.once("devModeReady", ({ registerPackageDebugFlag }) => {
 export class HEROSYS {
     static ID = "HEROSYS";
 
```

Comment:
2 thoughts to make the existence of `HEROSYS.module` slightly clearer (I'm not a fan of dynamically adding to objects for maintainability reasons although I appreciate that JavaScript has no problems with what you're doing):
1. `static module` - define it but don't initialize it explicitly at the point of definition or define it explicitly as `undefined`.
2. `static get module()` would get rid of brackets in a few places and give a perception that it should be read only. You'd then have a `_module` property or something that is defined like above that contains the actual value. Going a small step forward, presumably, you could have a write once `static set module()`.

Reply:
addressed in 0821f163c9d1fe8e53878f53c066a08ead020642

do you agree?